src/Spine/Data.hs

```diff
@@ -64,7 +68,27 @@ newtype TableName =
       renderTableName :: Text
     } deriving (Eq, Show)
 
+newtype IndexName =
+  IndexName {
+      renderIndexName :: Text
+    } deriving (Eq, Show)
+
+data SecondaryIndex = forall a.
```

Comment:
Wow, `ExistentialQuantification`. Might be nice to have a small comment about what and why.

Reply:
Will raise an issue to come back to do this, probably need a sweep of the entire project for more doc